examples/java/src/main/java/com/pelias/importer/BostonSchools.java

```diff
@@ -64,7 +64,7 @@ public void process() throws ClientProtocolException, IOException {
 
         // setup connection to Elasticsearch
         TransportClient elasticSearchClient = TransportClient.builder().build().addTransportAddress(
-                new InetSocketTransportAddress(InetAddress.getByName("localhost"), 9300));
+                new InetSocketTransportAddress(InetAddress.getByName("elasticsearch"), 9300));
```

Comment:
I'm just trying to get the thing working right now.  It works locally on 9300 but if I can get it working on 9200, I'll switch it back to that.